tests/integration/reqs/test_submit_multisigned.py

```diff
@@ -1,120 +1,127 @@
 from tests.integration.integration_test_case import IntegrationTestCase
-from tests.integration.it_utils import (
-    JSON_RPC_CLIENT,
-    sign_and_reliable_submission,
-    test_async_and_sync,
-)
+from tests.integration.it_utils import sign_and_reliable_submission, test_async_and_sync
 from tests.integration.reusable_values import WALLET
-from xrpl.core.binarycodec import encode_for_multisigning
-from xrpl.core.keypairs import sign
-from xrpl.ledger import get_fee
-from xrpl.models.amounts import IssuedCurrencyAmount
+from xrpl.asyncio.transaction.main import autofill, sign
+from xrpl.core.binarycodec.main import encode
 from xrpl.models.requests import SubmitMultisigned
-from xrpl.models.transactions import (
-    Signer,
-    SignerEntry,
-    SignerListSet,
-    TrustSet,
-    TrustSetFlag,
-)
+from xrpl.models.transactions import AccountSet, SignerEntry, SignerListSet
+from xrpl.transaction.multisign import multisign
+from xrpl.utils.str_conversions import str_to_hex
 from xrpl.wallet import Wallet
 
-FEE = get_fee(JSON_RPC_CLIENT)
-
-#
 # Set up signer list
-FIRST_SIGNER = Wallet.create()
-SECOND_SIGNER = Wallet.create()
+# Warning: Order of these seeds matter
+# Since SignerList will be sorted by codec
```

Comment:
What was the error you got when these were out of order? (If this was hard to debug, we should probably fix that because it's a very easy thing to do) - Maybe open a ticket with details?

Reply:
Yeah, the order really shouldn't matter...

Reply:
The SignerList got sorted by the codec when signing or something. This made it so `multisigned_tx.to_xrpl()` wasn't exactly what the ledger used for response data fields. Since it wasn't the same, `"tx_blob": encode(multisigned_tx.to_xrpl())` and `"hash": multisigned_tx.get_hash()` didn't match.

Reply:
I'm a little confused, the tx_blob and the hash shouldn't match because they have different data - the tx_blob is the encoded transaction and the hash is the hash of the transaction.

Reply:
I meant my expected tx_blob didn't match the actual tx_blob and the expected hash didn't match the actual hash. 

This was because when it submitted, it sorted the signer list and then filled in tx_blob and hash. But on my end, the transaction I submitted, was never sorted. So my calculated of expected values was not correct.

Reply:
You don't need to check that level of detail in the response - just make sure that it was successful and that it actually did change the domain. rippled does the checking for us.

Reply:
Oh, I thought it'd be better to have little more robust checking. I adapted bit of logic from xrpl.js submitMultisigned test. Do you suggest I change it to check less?
```
const expectedResponse: SubmitMultisignedResponse = {
      id: submitResponse.id,
      type: 'response',
      result: {
        engine_result: 'tesSUCCESS',
        engine_result_code: 0,
        engine_result_message:
          'The transaction was applied. Only final in a validated ledger.',
        tx_blob: multisigned,
        tx_json: {
          ...(decode(multisigned) as unknown as Transaction),
          hash: hashSignedTx(multisigned),
        },
      },
    }

    assert.deepEqual(submitResponse, expectedResponse)
   ```

Reply:
Yes. xrpl.js does more checking because it has response objects - xrpl-py does not, so it's unnecessary.

Reply:
Ok, removed lot of asserts and just checked domain and successful at end